packages/@n8n/config/src/configs/nodes.ts

```diff
@@ -0,0 +1,46 @@
+import { Config, Env, Nested } from '../decorators';
+
+function isStringArray(input: unknown): input is string[] {
+	return Array.isArray(input) && input.every((item) => typeof item === 'string');
+}
+
+class JsonStringArray extends Array<string> {
+	constructor(str: string) {
+		super();
+
+		let parsed: unknown;
+
+		try {
+			parsed = JSON.parse(str);
+		} catch {
+			return [];
+		}
+
+		return isStringArray(parsed) ? parsed : [];
+	}
+}
+
+@Config
+class CommunityPackagesConfig {
+	/** Whether to enable community packages */
+	@Env('N8N_COMMUNITY_PACKAGES_ENABLED')
+	enabled: boolean = true;
+}
+
+@Config
+export class NodesConfig {
+	/** Node types to load. Includes all if unspecified. @example '["n8n-nodes-base.hackerNews"]' */
+	@Env('NODES_INCLUDE')
+	readonly include: JsonStringArray = [];
```

Comment:
We still need to update `@Env` to create a new instance of this class during parsing, or this will always be `[]`.

Reply:
This works: https://share.cleanshot.com/KqprGFMB

Reply:
what JS sorcery is that 😅

Reply:
not sure how it works for you, but it definitely isn't working for me, in app or in tests 
![image](https://github.com/user-attachments/assets/ae593dbc-7a60-4a58-8923-5f138c181776)